cegs_portal/search/templates/search/index.html

```diff
@@ -1,6 +1,16 @@
 {% extends "base.html" %}
 {% load static i18n %}
 
+{% block css %}
+{{ block.super }}
+<link href="{% static 'css/nouislider.min.css' %}" rel="stylesheet">
+<style>
+    #hide-nav-searchbar {
```

Comment:
Can you format this a little nicer?

Reply:
![image](https://user-images.githubusercontent.com/19657615/224343050-4887f5c1-609c-49ce-81c2-3ab47a77fe86.png)

When I apply formatting it looks like this. Does this look sufficient or should I manually format it?

Reply:
That looks good

Reply:
Roger that